core/controllers/cron.py

```diff
@@ -38,6 +38,15 @@
 def require_cron_or_superadmin(handler):
     """Decorator to ensure that the handler is being called by cron or by a
     superadmin of the application.
+
+    Args:
+        handler: A handler instance
+
+    Returns:
+        handler: handler instance if it is authorized
+
+    Raises:
+        UnauthorizedUserException: Unauthorized user accesses this page
```

Comment:
There's no "page" here.

UnauthorizedUserException: An unauthorized user accesses the handler URL.